index.html

```diff
@@ -76,10 +76,12 @@ <h2 class="multi">Multi-Post Stories</h2>
     <p class="daily">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
         industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
     <div class="botones4">
-        <bUtton type="button" class="c">css</bUtton>
-        <bUtton type="button" class="h">html</bUtton>
-        <button type="button" class="b">bootstrap</button>
-        <button type="button" class="r">Ruby</button>
+        <ul>
+        <li><bUtton type="button" class="c">CSS</bUtton></li>
+       <li> <bUtton type="button" class="h">HTML</bUtton></li>
+       <li> <button type="button" class="b">BootStrap</button></li>
+       <li> <button type="button" class="r">Ruby</button></li>
+        </ul>
     </div>
     <div class="proyect">
         <button type="button" class="see">See Proyect</button>
```

Comment:
- [ ] (OPTIONAL) Good job styling your buttons 👍 but you have not implemented the various button interactions. It would be good to add this feature to all your buttons for a better user interface.
- Button interaction.
![Capture0](https://user-images.githubusercontent.com/85113567/154787171-c39bca0c-fcab-47fd-a4b4-0d3964ec002c.PNG)

Reply:
Yes apply the button interactions to all your buttons

Reply:
@latinogan also kindly include a pull request description which provides information about the changes you've made in this pull request

Reply:
coach are you able to check before i submit if every thing is ok

Reply:
@latinogan you've not still included the pull request description

Reply:
Hello @latinogan you have not added a description to your PR as recommended by the previous reviewers, however, I am willing to help with this. contact me @whoistolu